book/src/build_wasm.md

```diff
@@ -0,0 +1,74 @@
+# Building a WASM binary manually
```

Comment:
TODO(fzyzcjy): Double-check all resolved comments b/c desdaemon may have commented there but fzyzcjy did not see (comments get submerged)

Reply:
@fzyzcjy Would you like to take a brief look at the comments again before reviewing again?

Reply:
Yes, I should.